components/omnibox/browser/brave_omnibox_edit_model.cc

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/components/omnibox/browser/brave_omnibox_edit_model.h"
+
+#include "base/strings/string_util.h"
+
+#if !BUILDFLAG(IS_ANDROID) && !BUILDFLAG(IS_IOS)
```

Comment:
please replace all of these `!BUILDFLAG(IS_ANDROID) && !BUILDFLAG(IS_IOS)` guards with a dedicated buildflag for this feature